fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/DefaultScheduledDateGenerator.java

```diff
@@ -18,9 +18,10 @@
  */
```

Comment:
I have generally given the comments.... I will let other code experts to comment on sufficiency of the solution and also if there are any more problems with usecases